models/project_board.go

```diff
@@ -32,17 +33,19 @@ const (
 
 // ProjectBoard is used to represent boards on a project
 type ProjectBoard struct {
-	ID      int64 `xorm:"pk autoincr"`
-	Title   string
-	Default bool `xorm:"NOT NULL DEFAULT false"` // issues not assigned to a specific board will be assigned to this board
+	ID       int64 `xorm:"pk autoincr"`
+	Title    string
+	Default  bool `xorm:"NOT NULL DEFAULT false"` // issues not assigned to a specific board will be assigned to this board
+	Priority int
```

Comment:
The new column `priority` should have a migration

Reply:
@lunny ok I'm not super familiar with migrations, let me know if this works https://github.com/go-gitea/gitea/pull/12506/commits/2d433e1dac704b8002fcee669bdc3ca3426f0b67